src/v/kafka/server/tests/produce_consume_utils.h

```diff
@@ -45,8 +46,10 @@ struct kv_t {
             if (key_cardinality > 0) {
                 key = key % key_cardinality;
             }
-            records.emplace_back(
-              ssx::sformat("key{}", key), ssx::sformat("val{}", vstart + i));
+            const auto value = produce_tombstones
```

Comment:
similar to the comment in https://github.com/redpanda-data/redpanda/pull/23231/commits/c61ecff8e66e3c8a387aa31b15d5c151d8474f83#r1750158519

empty string value is not the same as tombstone (no value at all)